RequestService/RequestService.Repo/Repository.cs

```diff
@@ -1425,5 +1425,92 @@ public async Task<List<int>> UpdateRequestStatusToCancelledAsync(int requestId,
                 throw new Exception($"Error when updating request status to cancelled for requestId={requestId}");
             }
         }
+
+        private List<EntityFramework.Entities.Job> GetJobs(JobStatuses jobStatus)
```

Comment:
I'd suggest renaming this method to clarify that only jobs where start date has passed are returned